pkg/server/metrics.go

```diff
@@ -0,0 +1,96 @@
+package server
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/arquillian/ike-prow-plugins/pkg/log"
+	"github.com/arquillian/ike-prow-plugins/pkg/github/client"
+)
+
+// RegisterMetrics registers prometheus collectors to collect metrics
+func (s *Server) RegisterMetrics() []error {
```

Comment:
what is the receiver good for? there is nothing used here from the struct. It would make more sense to return an instance of the `Metrics` and then assign it in the caller...